backend/src/functions/Meetup/meetupRegistration.js

```diff
@@ -0,0 +1,33 @@
+import Services from '@/services';
+import Models from '@/models';
+import Middleware from '@/middleware';
+import { sendResponse } from '@/responses';
+import { ApiError } from '@/errors';
+
+const meetupRegistration = async (event) => {
+  const { id } = event.pathParameters;
+
+  if (!id || id.length !== 36) {
+    throw new ApiError(400, 'Meetup id missing or invalid');
```

Comment:
Should be 422 instead. the server was unable to process the request because it contains invalid data.